### PR TITLE
Update zhTW

### PR DESCRIPTION
**Pull request by BlueNightSky**
Roll back some translations and add new content.

### CODE REVIEW

**Comment by dmartinzar** [comment]
@mcc1 I'll let you decide whether this is good to merge. @BlueNightSky I know you are already in our Discord from the previous plugin, @mcc1 the link is here if you want to discuss in the #translators channel: https://discord.gg/EXCcmYW

**Comment by zarillion** [comment]
@mcc1 I'll let you decide whether this is good to merge. @BlueNightSky I know you are already in our Discord from the previous plugin, @mcc1 the link is here if you want to discuss in the #translators channel: https://discord.gg/EXCcmYW

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 1]
```diff
@@ -6,26 +6,25 @@ if not L then return end
...
-L["faeflayer_note"] = "瀑布後面隱藏的小洞穴內"
-L["gormtamer_tizo_note"] = nil
-L["humongozz_note"] = "種植一個 {item:175247} 在潮濕的沃土後將刷新 {npc:164122}。在一分鐘後 {spell:319367}，稀有將刷新。區域內的許多怪物都會掉落蘑菇"
+L["deifir_note"] = "騎上圈內的稀有並使用 {spell:319566} 和 {spell:319575} 使其減速和昏迷。"
+L["faeflayer_note"] = "瀑布後面隱藏的小洞穴內。"
+L["gormtamer_tizo_note"] = "擊殺 {npc:166735} 和 {npc:166736} 後騎著它們直到 {npc:164110} 刷新。"
+L["humongozz_note"] = "種植一個 {item:175247} 在潮濕的沃土將刷新 {npc:164122}。在一分鐘後 {spell:319367}，稀有將刷新。區域內的許多怪物都掉落蘑菇。"
 L["lehgo_note"] = "摧毀顫動的果姆蛋並擊殺 {npc:171827} 直到他出現。在洞穴內(入口在東南黃塵地穴內)"
-L["mymaen_note"] = nil
+L["mymaen_note"] = "擊殺區域內的腐楠精靈直到他做出表情並出現。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua:20]:
```suggestion
L["mymaen_note"] = "擊殺區域內的腐棘精靈直到他做出表情並出現。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 2]
```diff
@@ -414,11 +408,11 @@ L["options_icons_riftstone_desc"] = "顯示混亂裂隙石傳送者位置。"
 ----------------------------------- ICECROWN ----------------------------------
 -------------------------------------------------------------------------------
 
-L["prepatch_intro"] = "完成暗影之境前夕事件前置任務來解鎖此區域的首領"
-L["icecrown_rares"] = "所有首領以固定順序每20分鐘刷新一個.每個首領一天可以拾取多次"
-L["next_spawn"] = "下次刷新時間為本地時間 %s "
+L["prepatch_intro"] = "完成暗影之境前夕事件指引任務解鎖此區域的稀有。"
+L["icecrown_rares"] = "所有首領以固定的順序每20分鐘刷新一個。每個可以一天被拾取多次！"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["icecrown_rares"] = "所有首領以固定的順序每10分鐘刷新一個。每個可以一天被拾取多次！"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 3]
```diff
@@ -414,11 +408,11 @@ L["options_icons_riftstone_desc"] = "顯示混亂裂隙石傳送者位置。"
 ----------------------------------- ICECROWN ----------------------------------
 -------------------------------------------------------------------------------
 
-L["prepatch_intro"] = "完成暗影之境前夕事件前置任務來解鎖此區域的首領"
-L["icecrown_rares"] = "所有首領以固定順序每20分鐘刷新一個.每個首領一天可以拾取多次"
-L["next_spawn"] = "下次刷新時間為本地時間 %s "
+L["prepatch_intro"] = "完成暗影之境前夕事件指引任務解鎖此區域的稀有。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["prepatch_intro"] = "完成暗影之境前夕事件前置任務解鎖此區域的稀有。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 4]
```diff
@@ -377,10 +371,10 @@ L["dekaris_note"] = "在一個大型的突出岩石上。"
 L["ekphoras_note"] = "和3名其他玩家，站在平臺的角落並施放 {spell:330650} 召喚稀有。"
 L["ikras_note"] = "環繞墮滅堡飛行。這是個攻擊他的好位置。"
 L["orophea_note"] = "從東南方的拾取 {spell:337143} 並提供給 {npc:172577} 啟動。"
-L["sanngror_note"] = "如果他是不可攻擊，等待直到他不再對靈魂實驗。"
-L["talaporas_note"] = "和3名其他玩家，站在平臺的角落並施放 {spell:331800} 召喚稀有。"
-L["valis_note"] = "按正確順序點選三個 {npc:174810} 召喚稀有。每次嘗試順序都會更換，並且點選錯誤的符文會受到傷害和負面效果 {spell:343636}。"
-L["yero_note"] = "靠近 {npc:172862} 然後跟著他進入附近的一個山洞，之後他會變成敵對。"
+L["sanngror_note"] = "如果他是不可攻擊，等待直到他不再靈魂實驗。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["sanngror_note"] = "如果他是不可攻擊，等待直到他不再對靈魂做實驗。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 5]
```diff
@@ -350,20 +344,20 @@ L["sinrunner_note_day6"] = "從城牆破洞的 {npc:167815} 購買3個 {item:179
...
+L["sylla_note"] = "在這片令人毛骨悚然的地區，人們永遠無法期望在這裡展開一場出色的戰鬥，可惜我們在這裡。不要浪費我的時間。（暫）"
 
-L["grand_inquisitor_note"] = nil
-L["high_inquisitor_note"] = nil
-L["inquisitor_note"] = nil
-L["options_icons_inquisitors"] = nil
-L["options_icons_inquisitors_desc"] = nil
+L["grand_inquisitor_note"] = "把10個 {item:180451} 交給 {npc:160248} 有機會獲得此罪碑。"
+L["high_inquisitor_note"] = "把250個 {currency:1816} 交給 {npc:160248} 有機會獲得此罪碑。"
+L["inquisitor_note"] = "把100個 {currency:1816} 交給 {npc:160248} 有機會獲得此罪碑。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["inquisitor_note"] = "把100個 {currency:1816} 交給 {npc:160248} 有機會獲得此罪孽石。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 6]
```diff
@@ -350,20 +344,20 @@ L["sinrunner_note_day6"] = "從城牆破洞的 {npc:167815} 購買3個 {item:179
...
+L["eyegor_note"] = "艾戈爾准備作戰！（暫） "
+L["sylla_note"] = "在這片令人毛骨悚然的地區，人們永遠無法期望在這裡展開一場出色的戰鬥，可惜我們在這裡。不要浪費我的時間。（暫）"
 
-L["grand_inquisitor_note"] = nil
-L["high_inquisitor_note"] = nil
-L["inquisitor_note"] = nil
-L["options_icons_inquisitors"] = nil
-L["options_icons_inquisitors_desc"] = nil
+L["grand_inquisitor_note"] = "把10個 {item:180451} 交給 {npc:160248} 有機會獲得此罪碑。"
+L["high_inquisitor_note"] = "把250個 {currency:1816} 交給 {npc:160248} 有機會獲得此罪碑。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["high_inquisitor_note"] = "把250個 {currency:1816} 交給 {npc:160248} 有機會獲得此罪孽石。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 7]
```diff
@@ -350,20 +344,20 @@ L["sinrunner_note_day6"] = "從城牆破洞的 {npc:167815} 購買3個 {item:179
...
+L["addius_note"] = "頭腦虛弱的人不應浪費我的時間，但是如果您堅持，我會向您展示真正的痛苦。（暫）"
+L["eyegor_note"] = "艾戈爾准備作戰！（暫） "
+L["sylla_note"] = "在這片令人毛骨悚然的地區，人們永遠無法期望在這裡展開一場出色的戰鬥，可惜我們在這裡。不要浪費我的時間。（暫）"
 
-L["grand_inquisitor_note"] = nil
-L["high_inquisitor_note"] = nil
-L["inquisitor_note"] = nil
-L["options_icons_inquisitors"] = nil
-L["options_icons_inquisitors_desc"] = nil
+L["grand_inquisitor_note"] = "把10個 {item:180451} 交給 {npc:160248} 有機會獲得此罪碑。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["grand_inquisitor_note"] = "把10個 {item:180451} 交給 {npc:160248} 有機會獲得此罪孽石。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 8]
```diff
@@ -328,10 +322,10 @@ L["worldedge_gorger_note"] = [[
 L["dredglaive_note"] = "在 {npc:173671} 屍體的橋下。"
 L["forbidden_chamber_note"] = "在鎖著的門前拾取一個廢棄的靈魄之罐並學習 {spell:340701}。使用它吸取五個附近的 {npc:173838s}，然後在 {npc:173786} 前使用 {spell:340866}。"
 L["gilded_plum_chest_note"] = "擊殺在路上徘徊的 {npc:166680}。"
-L["lost_quill_note"] = "從禁忌圖書館的桌上瓶子拾取 {item:182475}。"
-L["rapier_fearless_note"] = "點選地上的劍杆，然後擊敗 {npc:173603}。"
+L["lost_quill_note"] = "從禁忌圖書館的桌上瓶子拾取 {item:182475}，然後把它交給外面拱門頂上的 {npc:173449}。"
+L["rapier_fearless_note"] = "點擊地上的劍杆，然後擊敗 {npc:173603}。"
 L["remlates_cache_note"] = "在暗隱村墓穴後面的外牆上。"
-L["smuggled_cache_note"] = nil
+L["smuggled_cache_note"] = ns.color.Red("*在拾取之前*：").."確認已完成 {quest:60480} 分支任務否則寶藏和任務將缺少40個 {currency:1820}。"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["smuggled_cache_note"] = ns.color.Red("*在拾取之前*：").."請確認已完成 {quest:60480} 支線任務否則寶藏和任務將缺少40個 {currency:1820}。"
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 9]
```diff
@@ -236,27 +230,27 @@ L["Oily"] = "油膩的無脊生物"
...
 L["runespeakers_trove_note"] = "東側找到 {npc:170563} 並擊殺他獲得 {item:181777}。"
 L["stolen_jar_note"] = "在巨大的蘑菇下面的山洞裡。"
 L["strange_growth_note"] = "攻擊 {npc:165037} 獲得寶藏。"
-L["vat_of_slime_note"] = "點選桌上的瓶子，然後點選軟泥桶。"
+L["vat_of_slime_note"] = "點擊桌上的瓶子，然後點擊軟泥桶。"
 
 L["giant_cache_of_epic_treasure"] = "巨大的史詩寶箱"
 L["spinebug_note"] = [[
-快看！{spell:343124}！勇敢的冒險家快過去，這肯定不是詭計。等一下，那是卡拉贊的音樂嗎？{npc:174663} 在這裡幹嘛…？
+快看！{spell:343124}！勇敢的冒險家快過去，這肯定不是詭計。等一下，那是卡拉贊的音樂嗎？{npc:174663} 在這裡干嘛…？
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
快看！{spell:343124}！勇敢的冒險家快過去，這肯定不是詭計。等一下，那是卡拉贊的音樂嗎？{npc:174663} 在這裡幹嘛…？
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 10]
```diff
@@ -236,27 +230,27 @@ L["Oily"] = "油膩的無脊生物"
...
 L["glutharns_note"] = "在軟泥瀑布後面的一個山洞裡。擊殺 {npc:172485} 和 {npc:172479} 解鎖寶箱。"
 L["kyrian_keepsake_note"] = "檢查 {npc:169664} 拾取寶藏。"
 L["misplaced_supplies"] = "在巨大的蘑菇頂上。跑上山後跳上較小的棕色蘑菇，在跑道較大的山丘後跳上巨大的蘑菇。"
 L["necro_tome_note"] = "要進入塔樓，必須完成一個 {npc:166657} 的小任務線。在頂層的書櫃裡。"
 L["plaguefallen_chest_note"] = [[
-站在綠色軟泥中(需要治療！)獲得10層 {spell:330069} 並會轉化為 {spell:330092}。
+站在綠色軟泥中（需要治療！）獲得10層 {spell:330069} 並會轉化為 {spell:330092}。
 
-轉化後，到 {npc:158406} 平臺(入口在東側)下面洞穴並點選管道傳送寶箱。
+轉化後，到 {npc:158406} 平台（入口在東側）下面洞穴並點擊管道傳送寶箱。
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
轉化後，到 {npc:158406} 平台（入口在東側）下面洞穴並點擊管線來傳送到寶箱。
```

File: plugins/09_Shadowlands/localization/zhTW.lua [hunk 11]
```diff
@@ -236,27 +230,27 @@ L["Oily"] = "油膩的無脊生物"
 
 L["blackhound_cache"] = "黑犬寶箱"
```
Comment (mcc1) [inline plugins/09_Shadowlands/localization/zhTW.lua]:
```suggestion
L["blackhound_cache"] = "黑犬儲物箱"
```

**Review by mcc1 — CHANGES_REQUESTED** [review]
update some translation. Thanks @adavak provide more data

**Pull request merged**